.github/workflows/build-test-deploy.yml

```diff
@@ -40,6 +40,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Fetch origin main
+        run: git fetch origin main
```

Comment:
Why is this needed? What problem does this solve?

Reply:
I'm trying to avoid several commits with changes to all java files. Current settings seems ok, but I'm sure we'll be amending them one or two times until Google format fits what we want.

Therefore I added `ratchetFrom("origin/main")` below, so spotless will fail only for the newly changed files.
However as we do shallow clone without origin/main this fails.

Reply:
>I'm trying to avoid several commits with changes to all java files

To be honest I would not try to avoid that at all. Just make a huge PR with changes to all Java files. Why not?

Reply:
Ok. I can do that